### PR TITLE
Edited homepage language

### PR DESCRIPTION
**Pull request by kbneedscoffee**
Edited language for minor grammar issues, clarity, and consistency

## Purpose

Came across Richie while looking for e-learning and LMS projects open to contributions. The website is very clean and well-designed. I am a UX writer and happy to help clarify language for end-users. Thus, I've edited the written content on the homepage to hopefully help people who find and want to use Richie in English.

## Changes

- Fix minor grammatical errors
- Fix passive voice (when appropriate)
- Language focused on potential users
- Some jargon was simplified, as website states Richie can be used by non-software engineers

### CODE REVIEW

File: website/pages/en/index.js [hunk 1]
```diff
@@ -71,8 +71,8 @@ module.exports = class Index extends React.Component {
         {[
           {
             content:
-              '<p>Richie is a specialized Content Management System designed to make learning portals. It is full-featured and ready-to-go.</p>' +
-              '<p>It is a toolbox to easily create full fledged websites with a catalog of online courses in days, not months.</p>' +
+              "<p>Richie helps educators create online learning portals. </p>" +
+              "<p>Build websites and online course catalogs in days with Richie's content management system.</p>" +
```
Comment (sampaccoud) [inline website/pages/en/index.js:75]: can we say like this?
```suggestion
              "<p>Build websites around online course catalogs in days with Richie's content management system.</p>" +
```
or
```suggestion
              "<p>Build websites including online course catalogs in days with Richie's content management system.</p>" +
```

File: website/pages/en/index.js [hunk 2]
```diff
@@ -117,9 +117,9 @@ module.exports = class Index extends React.Component {
         {[
           {
             content:
-              '<p>Richie is released under the MIT licence and all its documentation is copyleft. You can use and modify it as you please.</p>' +
-              '<p>It was built with open source tools such as Django CMS (& Django), and React. Everything is done in the open. Reach out and start contributing!</p>' +
-              `<a class="button" href="${siteConfig.repoUrl}">See the code</a>`,
+              '<p>Built with django CMS, Django, and React - everything is in the open. Richie is available under the MIT license.</p>' +
```
Comment (sampaccoud) [inline website/pages/en/index.js:120]:
```suggestion
              '<p>Built with Django CMS, Django, and React - everything is in the open. Richie is available under the MIT license.</p>' +
```

File: website/pages/en/index.js [hunk 3]
```diff
@@ -135,38 +135,32 @@ module.exports = class Index extends React.Component {
...
+                  such as Open edX or Moodle. Richie aggregates it all for your users.
                 </p>
-                <h3>An editor back-office</h3>
+                <h3>Author-first</h3>
                 <p>
-                  All the content in Richie is meant to be created and updated by regular editors,
-                  not software engineers. As it is based on Django CMS, you get a rich editor
-                  interface along with Richie.
+                  Content authors do not have to rely on software engineers to create and update all materials in Richie. 
+                  Instead, authors use a rich editor interface to maintain content.
```
Comment (sampaccoud) [inline website/pages/en/index.js:146]: should we mention "frontend editing"?

File: website/pages/en/index.js [hunk 4]
```diff
@@ -135,38 +135,32 @@ module.exports = class Index extends React.Component {
...
               </div>
             </div>
             <div className="blockElement twoByGridBlock">
               <div className="blockContent">
                 <h3>A multilingual website</h3>
                 <p>
-                  Richie itself is already available in more than one language, and you can add
-                  yours by talking to us. All the content can be added and managed in as many
-                  languages as you need.
+                  Richie itself is already available in more than one language, and you can add yours by talking to us. All the content can be added and managed in as many languages as you need. Richie is available in English, French, and Spanish. Want to add yours? Reach out! Richie supports content creation and management in as many languages as you need.
```
Comment (sampaccoud) [inline website/pages/en/index.js:159]: We added portuguese and russian. Should we list them?

File: website/pages/en/index.js [hunk 5]
```diff
@@ -180,11 +174,11 @@ module.exports = class Index extends React.Component {
         {[
           {
             content:
-              '<p>Stakeholders of the project have a regular video-chat meetup, in English. It takes place on (some) Thursdays in the afternoon, UTC time.</p>' +
-              "<p>If you're considering using or contributing to Richie, come introduce yourself and get some help!</p>",
+              '<p>Project stakeholders regularly check in through virtual meetups in English. Meetings typically take place on Thursday afternoons, UTC.</p>' +
```
Comment (sampaccoud) [inline website/pages/en/index.js:177]: it's now tuesdays afternoon :hand_over_mouth:

**Review by sampaccoud — COMMENTED** [review]
Wow this is awesome! Thank you so much for this contribution :pray: 



I made a few comments. What do you think?

File: website/pages/en/index.js (hunk 1 above)
Comment (kbneedscoffee) [reply website/pages/en/index.js:75]: The second option flows better.

File: website/pages/en/index.js (hunk 2 above)
Comment (kbneedscoffee) [reply website/pages/en/index.js:120]: 👍👍

File: website/pages/en/index.js (hunk 3 above)
Comment (kbneedscoffee) [reply website/pages/en/index.js:146]: It depends on the audience. If the "selling" point is that anyone unfamiliar with code can use Richie, mentioning "frontend editing" may be confusing.

File: website/pages/en/index.js (hunk 4 above)
Comment (kbneedscoffee) [reply website/pages/en/index.js:159]: Hm, I think that may be too many for easy reading. I would write "Richie is available in English, French, Spanish, and more." with perhaps a link to documentation on available languages.

File: website/pages/en/index.js (hunk 5 above)
Comment (kbneedscoffee) [reply website/pages/en/index.js:177]: 😳 Ah! Easy enough fix.

File: website/pages/en/index.js (hunk 4 above)
Comment (sampaccoud) [reply website/pages/en/index.js:159]: Perfect! You can point to : https://crowdin.com/project/richie

File: website/pages/en/index.js (hunk 3 above)
Comment (sampaccoud) [reply website/pages/en/index.js:146]: ok then. Let's leave it like this :+1:

File: website/pages/en/index.js (hunk 1 above)
Comment (sampaccoud) [reply website/pages/en/index.js:75]: :+1:

**Review by sampaccoud — APPROVED** [review]
Let's merge it :tada: 
Thank you again :pray:

**Comment by kbneedscoffee** [comment]
🙏 It looks like I can't merge due to permissions, please let me know if there's something I can do.

**Comment by sampaccoud** [comment]
@kbneedscoffee we have linting issues on the website. If you have yarn installed, you can solve it by running:
```bash
cd website
yarn prettier-write
```
And then committing the diff in the project.

**Comment by jbpenrath** [comment]
Hi @kbneedscoffee, many thanks for your contribution.
You also need to rewrite your commit message to respect [our commit convention](https://openfun.gitbooks.io/handbook/content/git.html).

An example of a valid message :
```
📝(website) edited homepage language 

Edited language for minor grammar issues, clarity, and consistency
```

**Comment by sampaccoud** [comment]
> Hi @kbneedscoffee, many thanks for your contribution. You also need to rewrite your commit message to respect [our commit convention](https://openfun.gitbooks.io/handbook/content/git.html).
> 
@kbneedscoffee Don't worry I'll do it while merging as we need to squash the 3 commits in one anyway.

**Comment by sampaccoud** [comment]
@kbneedscoffee Thanks again for your contribution. Would love to hear how you plan to use Richie and if/how we can make it better for your use case.

**Pull request merged**